distribution/dashboard/ui/src/pages/Tenant/Detail/Topo/index.tsx

```diff
@@ -1,4 +1,29 @@
+import { getNSName } from '@/pages/Cluster/Detail/Overview/helper';
+import TopoComponent from '@/pages/Cluster/Detail/Topo';
+import { getTenant } from '@/services/tenant';
+import { useRequest } from 'ahooks';
+import BasicInfo from '../Overview/BasicInfo';
 
-export default function Topo(){
-    return <h1>Topo</h1>
-}
\ No newline at end of file
+export default function Topo() {
+  const [ns, name] = getNSName();
+  const { data: tenantResponse } = useRequest(getTenant, {
+    defaultParams: [{ ns, name }],
+  });
+  const tenantTopoData = tenantResponse?.data;
+  return (
+    <div>
+      {tenantTopoData && (
+        <TopoComponent
+          tenantTopoData={tenantTopoData.replicas}
+          HeaderComp={
```

Comment:
HeaderComp is also a rare prop and can be changed to more common props such as header to reduce cognitive costs.